src/components/feedback/feedback.html

```diff
@@ -3,19 +3,67 @@
     <meta http-equiv="content-type" content="text/html; charset=utf-8" />
     <div style="height:30%"></div>
     <div class="modal-container" @click.stop style="height:70%;overflow-y:auto">
+        <span @click="close" class="close">&times;</span>
         <div class="modal-header">
-            <h2>Feedback</h2>
+            <div v-if="isFeedback" >
```

Comment:
using feedback component for messages as well feels a little wrong to me. If we're going to do it this way, then we should generify it a bit more. So have the title as a property for example. Essentially try and remove all the isFeedback branches.

Reply:
done